Extensions/xneko.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE XNeko **//
-//* VERSION 1.2 REV A **//
+//* VERSION 1.2.1 REV A **//
```

Comment:
This is not a valid version string. You either have the format `X.Y REV Z` or `X.Y.Z`
REV A = X.Y.0